src/tespy/components/heat_exchangers.py

```diff
@@ -2942,33 +2933,32 @@ def ttd_u_func(self):
         T_o2 = T_mix_ph(o2, T0=self.outl[1].T.val_SI)
         return self.ttd_u.val - T_bp_p(i1) + T_o2
 
-    dissipative = True
-    def exergy_balance(self, Tamb_val_SI, bus):
+    def exergy_balance(self, bus):
         r"""
         Calculate exergy balance of a condenser.
 
         Parameters
         ----------
-        Tamb_val_SI : float
-            Ambient temperature in K.
         bus  : tespy.connections.bus
             Energy flows in network. Used to calculate product exergy
             or fuel exergy of turbines and pumps.
-            
+
         Note
         ----
-            .. math::
+        .. math::
 
-                E_{P} = n/a
-                E_{F} = \dot{m}_{in} \cdot (e_{ph,in} - e_{ph,out})
+            E_{P} = \begin{cases}
+            nan & \text{heat dissipation (default)}
+            \dot{m}_{in,2} \cdot (e_{ph,in,2} - e_{ph,out,2}) & \\
+            \end{cases}\\
+            E_{F} = \dot{m}_{in} \cdot (e_{ph,in} - e_{ph,out})
         """
-        if self.dissipative == True:
-            self.E_P = 'n/a'
-        if self.dissipative == False:
-            self.E_P = self.inl[1].m.val_SI * (self.outl[1].ex_physical
-                                               - self.inl[1].ex_physical)
-        self.E_F = self.inl[0].m.val_SI * (self.inl[0].ex_physical
-                                           - self.outl[0].ex_physical)
+        if self.dissipative is True:
```

Comment:
How can I set dissipative to False?

Reply:
At the moment it would be something like `yourcondenser.dissipative = False` after simulation of the network and prior to the exergy analysis. Would be better, if it is just specified by the `set_attr()` method like every other parameter. I'll take this on.